app/dashboard/templates/bounty_details.html

```diff
@@ -75,6 +75,22 @@ <h5>Claimee Info</h5>
                             Github Profile: <span id="claimee_github_username"></span>
 
                         </div>
+                        <div id="interest_list">
+                            <br>
+                            <h5>Interested Profiles</h5>
```

Comment:
if we ever decide to do this with javascript, theres some examples if you search `text/x-jsrender`

Reply:
That makes sense.  I tend to drink the django kool-aid when it comes to handling data interaction with the template.